### PR TITLE
Error page created

### PR DESCRIPTION
**Pull request by wJoenn**
The error message will change depending on what's the error.
I also tried changing the back button from green to darkgrey to see how it's like as well as reducing the size of it a bit. What do you think ?

image

### CODE REVIEW

File: app/views/pages/error.html.erb [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+    <i class="fa-solid fa-arrow-left"></i>
+  <% end %>
+
+  <div class="error">
+    <h3>Oops<br/>That's unexpected</h3>
+    <%= image_tag "not_found.png" %>
+    <h1>No result was found</h1>
+    <% case @error %>
+    <% when "fetch_landmark_from_google_cloud_vision" %>
+      <p>The content of your photo was not recognised<br/>You cloud try again with another picture</p>
```
Comment (MarianaCarlota) [inline app/views/pages/error.html.erb:12]: *could

File: app/views/monuments/index.html.erb [hunk 2]
```diff
@@ -3,7 +3,7 @@
   data-controller="map"
   data-map-markers-value="<%= @markers.to_json %>"
   data-map-api-key-value="<%= ENV['MAPBOX_API_KEY'] %>">
-    <%= link_to root_path, class: "btn-back" do %>
+    <%= link_to root_path(ht: "y"), class: "btn-back" do %>
```
Comment (MarianaCarlota) [inline app/views/monuments/index.html.erb:6]: Maybe we should decide on one back button. The map has a different one and the forms have another. Maybe the one from Rutger will be more visible here

**Review by MarianaCarlota — CHANGES_REQUESTED** [review]

**Comment by MarianaCarlota** [comment]
Overall the page looks really good!

File: app/views/monuments/index.html.erb (hunk 2 above)
Comment (wJoenn) [reply app/views/monuments/index.html.erb:6]: You're right.
I'm not quite happy with the map's button as green , I can't really explain why but it looks kind of bland to me 🤔 
image

That's why I wanted to try another colour and see what you guys think about it
image

If everyone agrees on the green then I guess I'll just comply then, it's not that big of a deal.

Comment (Glod33) [reply app/views/monuments/index.html.erb:6]: What do you guys think about this one, it has a blur background.

Screenshot 2023-03-20 at 09 40 06

Comment (wJoenn) [reply app/views/monuments/index.html.erb:6]: I'd say introducing a new color that's only ever being used once would probably feel odd in the overall experience 🤔 
Maybe green text over darkgrey to have something similar as what you proposed ?
image

If not then let's just keep the green one, no point in staying fixated in a button's background for a day 😅

Comment (Glod33) [reply app/views/monuments/index.html.erb:6]: I think that's the best option, it's still a little unique and our style but indeed less bland

**Review by Glod33 — APPROVED** [review]

**Pull request merged**